client/plots/wsiviewer/WSIViewer.ts

```diff
@@ -13,6 +13,8 @@ import { WSIViewerInteractions } from '#plots/wsiviewer/interactions/WSIViewerIn
 import Settings from '#plots/wsiviewer/Settings.ts'
 import wsiViewerDefaults from '#plots/wsiviewer/defaults.ts'
 import { GetWSImagesRequest, GetWSImagesResponse } from '../../shared/types/routes/wsimages.ts'
+import wsiViewerImageFiles from '#plots/wsiviewer/images.ts'
```

Comment:
import from `./images.ts` instead?

Reply:
And also how about to rename images.ts to wsimagesloaded.ts.

Reply:
@aacic, The file name is changed to `wsimagesloaded.ts`